prover/Cargo.lock

```diff
@@ -7137,6 +7138,20 @@ dependencies = [
  "syn 2.0.48",
 ]
 
+[[package]]
+name = "zksync_protobuf_config"
+version = "0.1.0"
+dependencies = [
```

Comment:
this shouldn't be here

Reply:
It is part of the needed lock file update. The build with `--locked` fails without it.

Reply:
it shouldn't be needed once you removed the dependency from dal/Cargo.roml

Reply:
I guess I had some local discrepancy. It was fixed when I ran `cargo tree`.